Modules/_testcapi/run.c

```diff
@@ -78,14 +78,14 @@ run_fileexflags(PyObject *mod, PyObject *pos_args)
     result = PyRun_FileExFlags(fp, filename, start, globals, locals, closeit, pflags);
 
     if (closeit && result && _Py_IsValidFD(fd)) {
-        PyErr_SetString(PyExc_AssertionError, "File was not closed after excution");
+        PyErr_SetString(PyExc_AssertionError, "File was not closed after execution");
```

Comment:
This error message in a test module is not user-visible; it's not necessary to change it.

Reply:
revolved